### PR TITLE
Sort method

### PR DESCRIPTION
**Pull request by dariia-romanova**

### CODE REVIEW

**Review by JamesSunderlanddd — APPROVED** [review]
Good Job!

File: src/arrayMethodSort.js [hunk 1]
```diff
@@ -4,9 +4,38 @@
...
+
+    do {
+      countChanges = 0;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
+          const temp = this[i];
+
+          this[i] = this[i + 1];
+          this[i + 1] = temp;
```
Comment (ihor-jpeg) [inline src/arrayMethodSort.js:18]: as a useful thing to know, think (or google) how we can swap two values without a third variable. another useful thing would be figuring out how it works :)

**Review by ihor-jpeg — CHANGES_REQUESTED** [review]
The code works well. But there are a few things to work on.
When I run `[112, 44].sort2()`, your sorting method doesn't work. It returns [112, 44] instead of [44, 112]

**Review by ihor-jpeg — APPROVED** [review]
oh, I see. The default sort as strings was intended. My bad

File: src/arrayMethodSort.js (hunk 1 above)
Comment (dariia-romanova) [reply src/arrayMethodSort.js]: Changed to destructuring assignment)

**Pull request closed without merging**